src/Components/HeroSection.js

```diff
@@ -0,0 +1,61 @@
+import React, {useEffect} from 'react'
+import {Switch, Route} from "react-router-dom"
+import CartList from '../Pages/CartList'
+import ProductsList from '../Pages/ProductsList'
+import ProductDetail from '../Pages/ProductDetail'
+import WishList from '../WishList/WishList'
+import Home from '../Pages/Home'
+import axios from "axios"
```

Comment:
You can group all your exports from Pages folder in one index.js file .
Hence, the imports should look like this, 
```
import {ProductsList,cxartlist,productdetail,home }from '../Pages'
```